core/xmlparams.cpp

```diff
@@ -18,12 +18,17 @@ void xml_params_resize(struct xml_params *params, int count)
 
 void xml_params_add(struct xml_params *params, const char *key, const char *value)
 {
-	params->items.push_back({ std::string(key), std::string(value) });
+	xml_params_add(params, std::string(key), std::string(value));
+}
+
+void xml_params_add(struct xml_params *params, const std::string &key, const std::string &value)
+{
+	params->items.push_back({key, value});
 }
 
 void xml_params_add_int(struct xml_params *params, const char *key, int value)
 {
-	params->items.push_back({ std::string(key), std::to_string(value) });
+	params->items.push_back({std::string(key), std::to_string(value)});
```

Comment:
Albeit there is no explicit mention of space on the inside of curly brackets in https://github.com/subsurface/subsurface/blob/master/CODINGSTYLE.md, the examples are using space, as this is the most prevalent style used in our source.

Reply:
I don't think `.clang-format` is trying to be comprehensive in this project - there is a lot of 'use the pattern that is most commonly used in this project' going on.
In general, I think that introducing formatting changes in places where no actual code change is done, and where it is not a change to make the code more compliant with the coding style is creating additional effort for reviewing that has little value.

Reply:
> I'd assume that all desired code style requirements have been encoded in there?

Unfortunately not. This was written without C++ code in mind.